src/Adapter/Product/Validate/ProductValidator.php

```diff
@@ -133,6 +160,100 @@ private function validateShipping(Product $product): void
         $this->validateProductLocalizedProperty($product, 'delivery_out_stock', ProductConstraintException::INVALID_DELIVERY_TIME_OUT_OF_STOCK_NOTES);
     }
 
+    /**
+     * @param Product $product
+     *
+     * @throws ProductConstraintException
+     * @throws ProductPackConstraintException
+     * @throws ProductStockConstraintException
+     */
+    private function validateStock(Product $product): void
+    {
+        if ($this->advancedStockEnabled) {
+            $this->validateAdvancedStock($product);
+        } else {
+            $this->validateClassicStock($product);
+        }
+
+        $this->validateProductProperty($product, 'low_stock_threshold');
```

Comment:
Missing error codes.
according to https://github.com/PrestaShop/PrestaShop/pull/21110#discussion_r519180880
I think we came to conclusion that every prop should be validated and have dedicated error code.

Reply:
Yeah we decided this after I coded this, I'm gonna add them